src/enzo/QuantumGetUnits.C

```diff
@@ -0,0 +1,132 @@
+/***********************************************************************
+/
+/  COMPUTE AND RETURN THE UNITS FOR CALCULATING HBAR
+/
+/  written by: Xinyu Li
+/  date:       May, 2019
+/  modified1:
+/
+/  PURPOSE:  Returns the cosmology units:
+/
+/         time:        utim = 1 / sqrt(4 * \pi * G * \rho_0
+/         density:     urho = \rho_0 * (1+z)^3
+/         length:      uxyz = (1 Mpc) * box / h
+/         velocity:    uvel = uaye * uxyz / utim  (since u = a * dx/dt)
+/    (*)  temperature: utem = m_H * \mu / k * uvel**2
+/         a(t):        uaye = 1 / (1 + zri)
+/
+/           where:
+/             box     - size of simulation box in Mpc/h
+/             zri     - initial redshift (start of simulation)
+/             \rho_0  = 3*\Omega_0*H_0^2/(8*\pi*G)
+/             Omega_0 - the fraction of non-relativistic matter at z=0
+/
+/   NOTE: the units are calculated at a=1 (initial redshift)
+/
+************************************************************************/
+ 
+#include <stdio.h>
+#include <math.h>
+#include "ErrorExceptions.h"
+#include "macros_and_parameters.h"
+#include "CosmologyParameters.h"
+#include "units.h"
+#include "typedefs.h"
+#include "global_data.h"
+#include "phys_constants.h"
+#include "TopGridData.h"
+ 
+/* function prototypes */
+ 
+int CosmologyComputeExpansionFactor(FLOAT time, FLOAT *a, FLOAT *dadt);
+
+int QuantumCosmologyGetUnits(float *DensityUnits, float *LengthUnits,
```

Comment:
Sorry if I'm missing something, but is this function the same as`CosmologyGetUnits`?

Reply:
No, you are correct.  The whole QuantumGetUnits has been removed and FDM now uses the main GetUnits calls.